java/ql/lib/ext/java.lang.model.yml

```diff
@@ -8,6 +8,8 @@ extensions:
       - ["java.lang", "ClassLoader", True, "getSystemResource", "(String)", "", "Argument[0]", "path-injection", "ai-manual"]
       - ["java.lang", "ClassLoader", True, "getSystemResourceAsStream", "(String)", "", "Argument[0]", "path-injection", "ai-manual"]
       - ["java.lang", "Module", True, "getResourceAsStream", "(String)", "", "Argument[0]", "path-injection", "ai-manual"]
+      - ["java.lang", "Runtime", False, "load", "(String)", "", "Argument[0]", "command-injection", "ai-manual"]
+      - ["java.lang", "Runtime", False, "loadLibrary", "(String)", "", "Argument[0]", "command-injection", "ai-manual"]
```

Comment:
Although these would indeed cause vulnerabilities if user-controlled, I don't think they should be sinks of the command injection queries. Controlling which native library is going to be loaded is not the same as directly executing OS commands.

IMHO these should be sinks of a different query (just like the commented out rows at lines 32-33), but since that query doesn't exist yet, we probably shouldn't add them now.

Reply:
Thanks! I'll comment them out.